src/Application/Services/AskService.cs

```diff
@@ -49,7 +51,16 @@ public async Task<List<Ask>> GetItemAsks(Guid itemId)
                 .Where(x => x.ItemId == itemId)
                 .ToListAsync();
 
-            return asks;
+            if (!string.IsNullOrEmpty(_currentUserId))
+            {
+                asks.RemoveAll(x => x.CreatedBy == Guid.Parse(_currentUserId));
+                asks.ForEach(x =>
+                {
+                    x.Item.Asks.RemoveAll(x => x.CreatedBy == Guid.Parse(_currentUserId));
```

Comment:
Jak to się zachowa, gdy _currentUserId będzie nullem?

Reply:
Zwrócone zostaną wszystkie aski, tak jak było przed zmianą.
Gdy user jest zalogowany to ze zwracanych asków/bidów usuwane są jego własne, żeby nie było sytuacji, gdzie wyświetlany lowestAsk/highestBid to jego własny ask/bid
A gdy nie jest zalogowany to zwracane są wszystkie